obspy/core/frequency_domain_trace.py

```diff
@@ -88,9 +86,50 @@ def cross_correlation(self, tr2):
     def frequencies(self):
         n = self.data.size
         timestep = self.stats.delta
-        freq = fft.fftfreq(n, d=timestep)       
+        freq = fft.rfftfreq(n, d=timestep)       
         return freq
 
+    def cross_spectrum(self, tr2):
+        data1 = self.data
+        data2 = tr2.data
+        data2_conj = np.conj(data2)
+        cross_spectrum = data2_conj*data1
+        return (cross_spectrum)
+
+    def coherence(self, tr2):
+        data1 = self.data
+        data2 = tr2.data
+        data2_conj = np.conj(data2)
```

Comment:
Maybe a convenience property on FFTtrace would be useful..  `tr.conjugate` that returns `np.conjugate(tr.data)`?